### DISCUSSION TITLE
Retrieve domain associated with deleted team

### DISCUSSION THREAD

**Discussion by samuelfvlcastro** [vercel/community#3769] [Help] [answered]
### Summary

Hello,
I deleted the team (smashedbits) due to the project's lack of progress, but unfortunately I forgot to transfer the ownership of the domain I bought with the team (mockd.dev). Can you please help me transfer the domain to my personal account?

Thank you

### Steps to Reproduce

1. Create team
2. Create and deploy app associated with team
3. Purchase domain through team
4. Assign domain to deployed app in step 2
5. Transfer app to personal account
6. Delete team

**Comment by amyegan** [comment, marked as answer]
Hey @samuelfvlcastro. I found your domain registration associated with the deleted team and sent a transfer request. The domain will appear in your account after you accept the transfer.

**Reply by samuelfvlcastro** [reply]
Thank you very much,  it worked.

**Comment by Ethan-Hou1117** [comment]
@amyegan Having the same issue. Would it be possible for you to transfer the domain (www.rokobenefits.com) to my current team (Blitz)? My vercel user is ethan-hou1117. Thx!
